### ISSUE TITLE
How do I install python packages?

### ISSUE THREAD

**Issue by off99555** [20tab/UnrealEnginePython#456]
Hi. I'm very new to Unreal Engine 4. (Installed it yesterday)
So far I've only learned to use C++ programming with it very minimally.

I have a code in python that can detect face from image. I want to use this on UE4 to detect face from webcam.
My code in python requires external modules like `dlib` which can be installed via `pip install dlib`.
So how do I install `dlib` in UE4?
If not possible please suggest a topic I need to learn. It can be C++ too as `dlib` is also available in C++. But I simply don't know how to install these libraries from source onto UE4.

**Comment by fornof** [comment]
if you installed from binaries , you will have to point to your python path. 
Use that python to install your libraries. I was able to see 

> import numpy

  after I did 

> pip install numpy 

I'm guessing the same is for embedded, find the path, put in your packages , you're good to go.

**Comment by michaelStettler** [comment]
Hi, So I had the same idea using dlib and trying to use some landmarks to play around. Even though it seems in my case that the lib library is correctly imported, I am not able to use the .dat file that is used in dlib for the face landmarks. Did you made your model running ?

**Comment by off99555** [comment]
Instead of using python in any game engine, I send the data to external python code using ZeroMQ, networking library. It's simple and work to communicate between any languages. You should try it.

**Comment by fornof** [comment]
yeah, the only thing you're going to run into with zeromq is lag - it has to encode the info to binary, then decode it on the other side. If you're on a localhost , then everything is mostly okay I suppose. 1-2 ms lag. Decoding the info also takes time and I have noticed my frame rate changes from 60 fps to 40fps whenever I export a stream of data (in Blender)

**Comment by off99555** [comment]
@fornof Then you don't know how to use ZeroMQ properly. Because ZeroMQ is the technology that is very fast. You can even stream the video content from your PC to the phone wirelessly without much noticeable latency. Riftcat/Vridge is an example app that does this.

**Comment by KyoosikLee** [comment]
Hi do you guys have any idea installing python package for Android ?

**Comment by adahbingee** [comment]
goto directory `C:\Program Files\Epic Games\UE_4.27\Engine\Binaries\ThirdParty\Python3\Win64`
open powershell and input
```
curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
./python.exe get-pip.py
./Scripts/pip.exe install numpy
```
![image](https://user-images.githubusercontent.com/2985536/189030154-19e83612-635f-4f66-b2d8-9f81ce118914.png)
